source/_docs/automation/templating.markdown

```
@@ -1,30 +1,36 @@
 
-Each [trigger platform](/docs/automation/trigger/#event-trigger) can include additional data specific to that platform.
+Each [trigger](/docs/automation/trigger/#event-trigger) platform includes additional data specific to that {% term platform %}.
 
 ### All
 
-Triggers from all platforms will include the following data.
+Triggers from all platforms will include the following properties.
 
 | Template variable | Data |
 | ---- | ---- |
+| `trigger.platform` | Trigger object type.
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)